superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -113,9 +125,28 @@ const RightMenu = ({
             }
             icon={<Icons.TriangleDown />}
           >
-            {dropdownItems.map(
-              menu =>
-                findPermission(menu.perm, menu.view, roles) && (
+            {dropdownItems.map(menu => {
+              if (menu.label === 'Data') {
```

Comment:
For longer term use of this functionality, say if there's another submenu added later for example, do you think that instead of using the name of label as a condition, can we change the data structure of the menu to have the submenu within it? Then you could put the menuUploadLinks into the Data menu object and won't have to pass them in directly to the component, too. 
To explain in code: 
```
menu = [
  {
    label: "Data",
    .... (other key/values )
    subMenu: [
      {
         label: "Upload a CSV",
             .... (other key/values )
      }
}]
```
And then you can map and render recursively:
```
dropdownItems.map(menu => {
   if(!!menu.subMenu) {
       return (
        <SubMenu
         ....
       {menu.subMenu.map(item => (
......
}
```

Reply:
Yes I was thinking about this problem if we run into a case of a menu having multiple submenu's. As of now the fab api is returning these menu trees.  I think it would be smart to map out where everything should going foward on the frontend, especially if we have conditionals to where things should be placed.

Reply:
Also it would be nice to have the backend construct the submenu tree for cleanliness and ease but the fab api seems to only handle one level. So we might need to this for some time.

Reply:
Ok, I see what you're saying. It looks like the RightMenu component is largely hardcoded on the client side. Do we need to pass those submenu values in from the backend? We can add anything needed for toggling whether to show the items in the `FRONTEND_CONF_KEYS` if needed.